cluster-scope/base/core/namespaces/argocd/namespace.yaml

```diff
@@ -1,9 +1,8 @@
----
 apiVersion: v1
 kind: Namespace
 metadata:
   annotations:
-    openshift.io/display-name: "Operate First: ACM"
+    openshift.io/display-name: 'Operate First: Observatorium operator'
```

Comment:
This is not true

Reply:
That appears to be something you committed back in January.

```
$ git blame cluster-scope/base/namespaces/argocd/namespace.yaml
[...]
caf8ebe9 namespaces/base/observatorium-operator/namespace.yaml (<7453394+tumido@users.noreply.github.com> 2021-01-12 19:08:08 +0100 6)     openshift.io/display-name: "Operate First: Observatorium operator"
[...]
```

Reply:
Ah, interesting, `master` has both `namespaces/argocd.yaml` and `namespaces/acm.yaml`, and both directories have a `namespace.yaml` with the same namespace name defined.